docs/en-US/component/avatar.md

```diff
@@ -47,28 +47,28 @@ avatar/fit
-| `icon`    | representation type to icon, more info on icon component. | `string \| Component`                                      | —           | No       |
-| `size`    | avatar size.                                              | `number \| 'large' \| 'default' \| 'small'`                | `'default'` | No       |
-| `shape`   | avatar shape.                                             | `'circle' \| 'square'`                                     | `'circle'`  | No       |
-| `src`     | the source of the image for an image avatar.              | `string`                                                   | —           | No       |
-| `src-set` | native attribute `srcset` of image avatar.                | `string`                                                   | —           | No       |
-| `alt`     | native attribute `alt` of image avatar.                   | `string`                                                   | —           | No       |
-| `fit`     | set how the image fit its container for an image avatar.  | `'fill' \| 'contain' \| 'cover' \| 'none' \| 'scale-down'` | `'cover'`   | No       |
+| Name    | Description                                               | Type                                                              | Default     |
+| ------- | --------------------------------------------------------- | ----------------------------------------------------------------- | ----------- |
+| icon    | representation type to icon, more info on icon component. | ^[string] / ^[Component]                                          | —           |
+| size    | avatar size.                                              | ^[number] / ^[enum]`'large' \| 'default' \| 'small'`              | `'default'` |
+| shape   | avatar shape.                                             | ^[enum]`'circle' \| 'square'`                                     | `'circle'`  |
+| src     | the source of the image for an image avatar.              | `string`                                                          | —           |
+| src-set | native attribute `srcset` of image avatar.                | `string`                                                          | —           |
+| alt     | native attribute `alt` of image avatar.                   | `string`                                                          | —           |
+| fit     | set how the image fit its container for an image avatar.  | ^[enum]`'fill' \| 'contain' \| 'cover' \| 'none' \| 'scale-down'` | `'cover'`   |
 
-### Avatar Events
+### Events
 
-| Name    | Description                    | Type                 |
-| ------- | ------------------------------ | -------------------- |
-| `error` | trigger when image load error. | `(e: Event) => void` |
+| Name  | Description                    | Type                            |
+| ----- | ------------------------------ | ------------------------------- |
+| error | trigger when image load error. | ^[function]`(e: Event) => void` |
```

Comment:
function => Function
sry, this is my mistake.

Reply:
@chenxch done, PTAL :bow: